devenv/README.md

```diff
@@ -1,37 +1,39 @@
-This folder contains useful scripts and configuration for...
+# Dev environment setup
 
-* Configuring dev data sources in Grafana
-* Configuring dev & test scenarios dashboards.
-* Creating docker-compose file with DBs and fake data.
+This folder contains useful scripts and configuration so you can:
 
+* Configure dev data sources in Grafana
+* Configure dev and test scenario dashboards.
```

Comment:
Configure dashboards for development, and test scenarios.

Reply:
The presence of the comma depends on what you want to say here.

Are we configuring dashboards for development scenarios and test scenarios? If so, no comma.

If we are configuring dashboards for development and testing scenarios, then we could leave the comma, but probably should put the bit about testing scenarios in a different bullet.

Reply:
Sounds like no comma then, but I'm going to trust your judgement on this one 👍